### PR TITLE
Add reliability to DBotPredictURLPhishing

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-6137)

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by DinaMeylakh** [comment]
Uploading for review now, will add unittests and documentation next week. 
Checked the correction on xsoar and it works :) 
@ShahafBenYakir you can review now or wait until finaliziation

File: Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.yml [hunk 1]
```diff
@@ -59,6 +59,19 @@ args:
   - 'True'
   required: false
   secret: false
+- auto: PREDEFINED
+  description: Reliability of the source providing the intelligence data.
+  defaultValue: A+ - 3rd party enrichment
```
Comment (ShahafBenYakir) [inline Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.yml:64]: Let's validate with @bakatzir that this indeed should be the default

File: Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py [hunk 2]
```diff
@@ -761,6 +763,16 @@ def main():
         email_html = demisto.args().get('emailHTML', "")
         max_urls = int(demisto.args().get('maxNumberOfURL', 5))
         urls_argument = demisto.args().get('urls', '')
+        reliability = demisto.args().get("reliability", DBotScoreReliability.A_PLUS)
+
+        if DBotScoreReliability.is_valid_type(reliability):
+            reliability = DBotScoreReliability.get_dbot_score_reliability_from_str(
+                reliability
```
Comment (ShahafBenYakir) [inline Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py:770]: I don't understand the combination of this two functions, why do we need get_dbot_score_reliability_from_str?

**Review by ShahafBenYakir — COMMENTED** [review]
Great!
let's also not forget to add these to docs

File: Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.yml (hunk 1 above)
Comment (bakatzir) [reply Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.yml:64]: indeed, A+.

File: Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py (hunk 2 above)
Comment (DinaMeylakh) [reply Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py]: you are right, this is duplication. I removed one handling of invalid reliability.

File: Packs/PhishingURL/ReleaseNotes/1_1_0.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### DBotPredictURLPhishing
+
+- Added an optional 'reliability' argument to indicate the reliability of the source providing the intelligence data.
```
Comment (ShirleyDenkberg) [inline Packs/PhishingURL/ReleaseNotes/1_1_0.md:6]:
```suggestion
Added an optional 'reliability' argument to indicate the reliability of the source providing the intelligence data. 
```

File: Packs/PhishingURL/Scripts/DBotPredictURLPhishing/README.md [hunk 4]
```diff
@@ -11,9 +11,16 @@ This script uses the [Rasterize](https://xsoar.pan.dev/docs/reference/integratio
...
-| Tags |  |
+| Tags | ml |
 | Cortex XSOAR Version | 6.0.0 |
 
+## Used In
+
+---
+This script is used in the following playbooks and scripts.
+
+* Phishing - Machine Learning Analysis
```
Comment (ShirleyDenkberg) [inline Packs/PhishingURL/Scripts/DBotPredictURLPhishing/README.md:22]:
```suggestion
Phishing - Machine Learning Analysis
```

**Comment by ShirleyDenkberg** [comment]
@bakatzir @ShahafBenYakir Doc review completed.

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
We have decided to wait with the docker update for now, and so I am ignoring the docker validation error.

**Pull request merged**